microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -6,14 +6,14 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-{product-title-first} provides developers and IT organizations with small-form-factor and edge computing, delivered as an application that customers can deploy on top of their managed {op-system-base-full} devices at the edge. Built on {OCP} and Kubernetes, {microshift-short} provides an efficient way to operate single-node clusters in low-resource edge environments.
+{product-title-first} provides developers and IT organizations with small-form-factor and edge computing delivered as an application that customers can deploy on top of their managed {op-system-base-full} devices at the edge. Built on {OCP} and Kubernetes, {microshift-short} provides an efficient way to operate single-node clusters in low-resource edge environments.
 
 {microshift-short} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes consist of simply staging another version in parallel and then - without relying on a network - flipping to and from that version and restarting.
 
 [id="microshift-4-15-about-this-release"]
 == About this release
 // TODO: Update with the relevant information closer to release.
-Version 4.15 of {microshift-short} includes new features and enhancements. {microshift-short} was introduced as Generally Available with {microshift-short} 4.14. Update to the latest version of {microshift-short} to receive all of the latest features, bug fixes, and security updates. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with the CRI-O container runtime. New features, changes, and known issues that pertain to {product-title} {microshift-short} are included in this topic.
+Version 4.15 of {product-title} includes new features and enhancements. {microshift-short} was introduced as Generally Available with {microshift-short} 4.14. Update to the latest version of {microshift-short} to receive all of the latest features, bug fixes, and security updates. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with the CRI-O container runtime. New features, changes, and known issues that pertain to {microshift-short} are included in this topic.
```

Comment:
From what I see in our files on `release-4.15` branch, we're using 1.28.5 kubernetes

https://github.com/openshift/microshift/blob/release-4.15/Makefile.kube_git.var#L3

Reply:
But I don't think we want the patch version - it will change in future and that would be doc maintenance effort - 1.28 is sufficient?